libs/design/src/molecules/paginator/paginator.component.html

```diff
@@ -0,0 +1,14 @@
+<span (click)="onNotifyPrevPageChange()" *ngIf="showPrevPageChange()" class="daff-paginator__prev daff-paginator__element">Prev</span>
+
+<span class="daff-paginator__elipsis" *ngIf="showFirstElipsis()"><span class="daff-paginator__element" (click)="onNotifyPageChange(1)" [class.daff-paginator__element--selected]="isSelected(1)">1</span>...</span>
```

Comment:
`daff-paginator__page-link` instead of `daff-paginator__element`. And does this need to be an `<a>` instead of `<span>`? If it does, it'll affect styling.

Reply:
Sure that's fine. Done